bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/base/PowerLawSpectrum.java

```diff
@@ -24,16 +24,11 @@ public PowerLawSpectrum(final double powerLawIndex, final double start, final do
         _start /= (1 + z);
         _end /= (1 + z);
 
-        final int n = (int) ((_end - _start) / _sampling + 1);
+        final int n = (int) Math.round((_end - _start) / _sampling + 1);
```

Comment:
👍

Reply:
I don't think there should be any need to cast to `int` after `Math.round`, which, in this case, would return a `long`. We could just make `i` a `long` and then proceed as is being done.

Reply:
I think `n` must be an integer since it is used to define the size of `fluxArray` in line 28?

Reply:
You're right. I didn't know that you were obligated to use an `int` for size when creating a new array, but apparently, you are. 😊